File: LinkedList/1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        vector<bool> v(26,0);
        for(char c: allowed) v[c-'a'] = 1;
        int ans = 0;
        for(string& s: words){
            bool flag = 1;
            for(char c : s){
                if(v[c-'a'] == 0){
                    flag = 0;
                    break;
                }
            }
            if(flag) ans++;
        }
        return ans;
    }
};

File: LinkedList/1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution(object):
    def countConsistentStrings(self, allowed, words):
        """
        :type allowed: str
        :type words: List[str]
        :rtype: int
        """
        allowed = set(allowed)
        ans = 0
        for word in words:
            flag = 1
            for c in word:
                if c not in allowed:
                    flag = 0
                    break
            if flag == 1:
                ans += 1
        return ans; 

        